llm-servers/vllm/cpu/Containerfile

```diff
@@ -0,0 +1,29 @@
+# This vLLM Dockerfile is used to construct image that can build and run vLLM on x86 CPU platform.
+
+FROM registry.access.redhat.com/ubi9/python-311 AS cpu-setup
+
+USER 0
+
+RUN pip install --upgrade pip \
+    && pip install wheel packaging ninja "setuptools>=49.4.0" numpy
+
+FROM cpu-setup AS build
+
+WORKDIR /workspace
+RUN git clone https://github.com/RHRolun/vllm
+WORKDIR /workspace/vllm
+RUN git checkout 04e585a
```

Comment:
any reason for setting it up for this? perhaps worth a comment here about the reason

Reply:
It builds vllm from the repo, so figured easiest way to do that would be to git pull the repo during build-time, will add a comment

Reply:
I meant why checking out that specific commit? any reason for that? (if that so, add a comment explaining it) Perhaps better to do a tag/branch?

Reply:
Aah got it, I check out the specific commit to make it properly versioned. A tag would better.